src/fs.rs

```diff
@@ -475,12 +475,12 @@ impl<C: StaticFileConfig> Responder for NamedFile<C> {
 /// A helper created from a `std::fs::File` which reads the file
 /// chunk-by-chunk on a `CpuPool`.
 pub struct ChunkedReadFile {
-    size: u64,
-    offset: u64,
-    cpu_pool: CpuPool,
-    file: Option<File>,
-    fut: Option<CpuFuture<(File, Bytes), io::Error>>,
-    counter: u64,
+    pub size: u64,
```

Comment:
What is the reason for this change?

Reply:
ChunkedReadFile is public and shows up in the documentation, but it's not useable, because it's fields are private.

Reply:
So, are you okay with this change being included in the pull request?